chocolatey/choco-theme
# (#398) Ensure Disqus works in dark mode

## Description Of Changes
Due to the Bootstrap upgrade to 5.3, Disqus
comments were not readable in dark mode. This
issue was reported and confirmed at:

https://disqus.com/home/discussion/channel-discussdisqus/embed_disqus_comments_not_changing_background_color_with_bootstrap_53_color_mode/

A change has been made to override the
color-scheme at the :root, which appears not to
affect anything else in choco-theme currently.

In addition, the Disqus JavaScript has been moved
into it's own file so that the code can be shared
across repositories.

## Motivation and Context
We need these comments to be readable.

## Testing
1. Run the site and go to the Disqus comment section while in dark. mode
2. Everything should appear readable
3. Switch to light mode without reloading the page
4. Everything should appear readable

### Operating Systems Testing
n/a

## Change Types Made

## Change Checklist


## Related Issue
#398
